bot/cogs/utilities.py

```diff
@@ -138,6 +140,15 @@ async def version(self, ctx: Context) -> str:
 
         return msg
 
+    @command(brief="[math expression]", aliases=["m"])
+    async def math(self, ctx: Context, infix_expression: str) -> None:
+        """Evaluates a mathmatical infix expression with the shunting yard algorithm."""
+        result: Union[float, int] = evaluate(infix_expression)
```

Comment:
If the evaluation fails this will raise an error, I think this should be handled here to present the user with a tailored error message.